server/database/queries/users/getUsers/index.js

```diff
@@ -0,0 +1,10 @@
+const connection = require('../../../data/connection');
+
+const getAllUsers = () => {
+  const sql = {
+    text: 'SELECT * FROM users',
```

Comment:
This query returns everything in the user's columns even the password and that's not a thing we want to return a response with